tests/unit/test_charm_2.py

```diff
@@ -315,8 +322,9 @@ def test_core_reconciliation_before_peer_relation_ready(self):
         """
         self.harness.begin_with_initial_hooks()
 
-        self.assertFalse(
-            self.harness.charm._core_reconciliation(),
+        self.assertRaises(
+            WordPressWaitingStatusException,
+            lambda _: self.harness.charm._core_reconciliation(),
```

Comment:
Is the lambda needed here? Couldn't `self.harness.charm._core_reconciliation` be passed directly since it doesn't take any arguments?

Reply:
Fixed, thanks!